File: TherapyUI/src/app/models/Appointment.ts
import { AppointmentType } from "./AppointmentType";
import { Client } from "./Client";
import { Therapist } from "./Therapist";

export class Appointment {
    id: string;
    appointmentType: AppointmentType;
    client: Client;
    therapist: Therapist
    appointmentDate: Date;
}

File: TherapyAPI.backup/Models/Appointment.cs
using System;

namespace TherapyAPI.Models {
    public class Appointment {
        public Guid Id { get; set; }
        public AppointmentType AppointmentType { get; set; }
        public Client Client { get; set; }
        public Therapist Therapist { get; set; }
        public DateTime AppointmentDate { get; set; }
    }
}